docs/userGuide/contentAuthoring.md

```diff
@@ -113,6 +113,34 @@ In addition, it supports:
 
 * More media blocks, embedding services and additional options can be found in [Markdown-it-block-embed docs](https://github.com/rotorz/markdown-it-block-embed)
 
+### Use MarkBind variables
+
+While authoring content, you can insert variables to be replaced by pre-defined values anywhere in your site. MarkBind treats any text surrounded with double curly braces as a variable, e.g. <code>{<span></span>{variable_name}}</code>.
+
+Some helpful variables are provided by MarkBind for your convenience, as shown below. 
+
+#### Glyphicons
+
+This asset is provided by [Glyphicons](https://glyphicons.com/) via Bootstrap.
+<br><br>
+<tip-box>
+Traditional usage:  
+`<span class='glyphicon glyphicon-hand-right' aria-hidden='true'></span>`
+<br>  
+MarkBind usage:  
+<code>{<span></span>{glyphicon-hand-right}}</code>
+</tip-box>
+
+Use these [available glyphs](https://getbootstrap.com/docs/3.3/components/#glyphicons) on your site by surrounding a glyph's name with double curly braces.  
+e.g. <code>{<span></span>{glyphicon_hand_right}}</code> will be rendered as <code><span class='glyphicon glyphicon-hand-right' aria-hidden='true'></span></code>
+
+Note: Replace all instances of `-` with `_` in the glyph's name, e.g. <code>{<span></span>{glyphicon-hand-right}}</code> becomes <code>{<span></span>{glyphicon_hand_right}}</code>

```

Comment:
Full stop at the end.

Reply:
The rest of the page doesn't use a full stop at the end.